### PR TITLE
feat: Add Monitor Downtime Docs

### PR DESCRIPTION
**Pull request by marcusperezNR**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by rhetoric101** [comment]
@marcusperezNR could you sign our CLA again? It seems to have forgotten you had done this.

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx [hunk 1]
```diff
@@ -1018,3 +1018,249 @@ You use [secure credentials](/docs/synthetics/synthetic-monitoring/using-monitor
   ```
   </Collapser>
 </CollapserGroup>
+
+## Manage your monitor downtimes [#monitor-downtimes]
+
+A [monitor downtime](/docs/synthetics/synthetic-monitoring/using-monitors/monitor-downtimes-disable-monitoring-during-scheduled-maintenance-times) allows you to specify times that your synthetic monitors will stop running during their specified period.
+Example timezone values can be found here, under the "TZ Identifier" section: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```
Comment (akristen) [inline src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx:1025]: @marcusperezNR Can you provide an example of how the timezones should be formatted for the value in the code snippets? For example, maybe two examples: one for North America and a second for a timezone in EMEA? 

We don't want to post raw links like this, and the wikipedia page is so dense that it doesn't give customers much direction about how their values should appear in code.

Comment (marcusperezNR) [reply src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx]: For sure! A couple examples are:

1. MST: "America/Phoenix"
2. CEST: "Europe/Berlin"

For this portion of the docs, we were looking to emulate what alerts does [here](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications/#:~:text=timeZone%3A%20The%20time%20zone%20that%20applies%20to%20the%20muting%20rule%20schedule.%20Example%3A%20%27America/Los_Angeles%27.%20See%20Wikipedia%27s%20list%20of%20tz%20database%20time%20zones.), but I realize theirs is quite a bit more clear than ours, so I can edit it to look more like that! Please just lmk

Comment (marcusperezNR) [reply src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx]: I updated that line to better reflect what alert muting rules has!

**Review by akristen — APPROVED** [review]
Thanks for working with us on this! I've updated the language but this otherwise looks good.

**Pull request merged**